src/mmw/mmw/settings/layer_settings.py

```diff
@@ -145,6 +145,30 @@
         # overlaps with perimeter polygon.
     },
     {
+        'code': 'nhd_quality_tp',
+        'display': 'Delaware River Basin TP Concentration',
```

Comment:
re: Layer Selector text: this isn't ideal, but we're currently handling the label for the "Delaware River Basin High Resolution Stream Network" by including a `<br />` tag and some spaces as the value for `display` here.